### DISCUSSION TITLE
Dependency on System.Runtime.Caching.dll

### DISCUSSION THREAD

**Discussion by enfJoao** [2sic/2sxc#3512] [General]
When cleaning my Oqtane install for the .Net 9 update, I'm getting a
HTTP Error 500.30 - ASP.NET Core app failed to start
when removing System.Runtime.Caching.dll

Shawn mentioned that Oqtane 6 and .Net9 does not rely on this library, asking if there are any other modules installed.
https://github.com/oqtane/oqtane.framework/issues/4848

The only one I have is 2sxc, so does it depend on System.Runtime.Caching.dll?

**Comment by iJungleboy** [comment]
2sxc requires this.

The latest release includes it in the installation; previous releases did not, as it was previously part of Oqtane.
